flowauth/backend/tests/test_version_route.py

```diff
@@ -5,9 +5,8 @@
 import flowauth
 
 
-def test_version(client):
+def test_version(client, app):
```

Comment:
not sure why you add `app` here but don't use it.

Reply:
I think that importing the fixture also provides some setup

Reply:
I took it out and it broke the test :sweat_smile: so I'll leave it in